src/pf-tabs/index.html

```diff
@@ -12,13 +12,36 @@
       <h1>Tabs</h1>
     </div>
     <pf-tabs>
-      <pf-tab tabTitle="Tab1" active="true">
-        <p>Tab1 content here</p>
+      <pf-tab tabTitle="Tab One" active="true">
+        <p>Tab one content here</p>
       </pf-tab>
-      <pf-tab tabTitle="Tab2">
-        <p>Tab2 content here</p>
+      <pf-tab tabTitle="Tab Two">
+        <p>Tab two content here</p>
       </pf-tab>
     </pf-tabs>
+
+    <pf-tabs>
+      <pf-tab tabTitle="Tab One" active="true">
+        <pf-tabs class="nav nav-tabs nav-tabs-pf">
+          <pf-tab tabTitle="Secondary Tab One" active="true">
+            <p>Secondary tab one content.</p>
+          </pf-tab>
+          <pf-tab tabTitle="Secondary Tab Two">
+            <p>Secondary tab two content.</p>
+          </pf-tab>
+        </pf-tabs>
+      </pf-tab>
+      <pf-tab tabTitle="Tab Two">
+        <pf-tabs class="nav nav-tabs nav-tabs-pf">
+          <pf-tab tabTitle="Secondary Tab One" active="true">
+            <p>Secondary tab one content.</p>
```

Comment:
If the demo could have different nested tab labels and content under the different main tabs that might help highlight functionality. Right now it appears slightly confusing, as if the tabs aren't functioning properly